BraveShared/tr.lproj/BraveShared.strings

```diff
@@ -0,0 +1,1353 @@
+
+/* The cancel button when confirming clear private data. */
+"ClearPrivateDataAlertCancelButtonTitle" = "İptal";
+
+/* No comment provided by engineer. */
+"ClearPrivateDataAlertMessage" = "Emin misiniz?";
+
+/* The button that clears private data. */
+"ClearPrivateDataAlertOkButtonTitle" = "Tamam";
+
+/* No comment provided by engineer. */
+"ClearPrivateDataAlertTitle" = "Verileri Temizle";
+
+/* No comment provided by engineer. */
+"ClearPrivateDataAlertYesAction" = "Evet, Sil";
+
+/* Button in settings that clears private data for the selected items. */
+"ClearPrivateDataNow" = "Verileri Şimdi Temizle";
+
+/* The cancel button when confirming clear history. */
+"ClearSyncedHistoryAlertCancelButtoTitle" = "İptal";
+
+/* Description of the confirmation dialog shown when a user tries to clear history that's synced to another device. */
+"ClearSyncedHistoryAlertMessage" = "Bu eylem, senkronize edilmiş cihazlarınızdan geçmiş de dahil olmak üzere tüm özel verilerinizi temizler.";
+
+/* The confirmation button that clears history even when Sync is connected. */
+"ClearSyncedHistoryAlertOkButtoTitle" = "Tamam";
+
+/* Button title to close a menu. */
+"Close" = "Kapat";
+
+/* No comment provided by engineer. */
+"CloseAllTabsTitle" = "Tüm %i Sekmelerini Kapat";
+
+/* Code words input help */
+"CodeWordInputHelp" = "Sağlanan eşitleme zinciri kod sözcüklerinizi aşağıdaki forma yazın.";
+
+/* Code words section title */
+"CodeWords" = "Kod Sözcükleri";
+
+/* Text to choose for downloading a file (example: saving an image to phone) */
+"CommonDownload" = "indir";
+
+/* Text to select sharing something (example: image, video, URL) */
+"CommonShare" = "Paylaş";
+
+/* No comment provided by engineer. */
+"Confirm" = "Onayla";
+
+/* Cookie settings option title */
+"CookieControl" = "Çerez Denetimi";
+
+/* Settings item for clearing cookies and site data */
+"Cookies" = "Çerezler ve Site Verileri";
+
+/* Copied action complete title */
+"Copied" = "Kopyalandı";
+
+/* Copied codewords title */
+"CopiedToClipboard" = "Panoya kopyalandı!";
+
+/* Copy app info to clipboard action sheet action. */
+"CopyAppInfoToClipboard" = "Uygulama bilgilerini panoya kopyalayın.";
+
+/* Context menu item for copying an image to the clipboard */
+"CopyImageActionTitle" = "Resmi Kopyala";
+
+/* Context menu item for copying a link URL to the clipboard */
+"CopyLinkActionTitle" = "Bağlantıyı Kopyala";
+
+/* Copy codewords title */
+"CopyToClipboard" = "Panoya kopyala";
+
+/* Copy rewards internals info for support */
+"CopyWalletSupportInfo" = "Destek Bilgilerini Kopyala";
+
+/* Create PDF share action. */
+"CreatePDF" = "PDF Oluştur";
+
+/* Currently usedd search engines section name. */
+"CurrentlyUsedSearchEngines" = "Şu anda kullanılan arama motorları";
+
+/* Title for quick-search engines settings section. */
+"CustomSearchEngines" = "Özel Arama Motorları";
+
+/* Data Saved Shield Stat */
+"DataSavedStat" = "Tah. Veri \nKaydedilmiş";
```

Comment:
Here the phrase has a meaning like we are **recording** data. It is very misleading. Better too use

"DataSavedStat" = "Tah. Veri \nMiktarı"; which is in in English like "Estimated Data Amount"